test/models/postcode_local_restriction_search_test.rb

```diff
@@ -84,6 +84,21 @@
     end
   end
 
+  describe "#no_restriction_information_england_postcode?" do
```

Comment:
Could we add a test for the country that isn't England?

Reply:
have added!